services/src/atdd/src/test/java/org/openkilda/atdd/FlowFfrTest.java

```diff
@@ -48,7 +49,7 @@
 import javax.ws.rs.client.Entity;
 import javax.ws.rs.core.Response;
 
-public class FlowFFRTest {
+public class FlowFfrTest {
     private static final int DEFAULT_DISCOVERY_INTERVAL = 10;
     private static final String sourceSwitch = "00:00:00:00:00:00:00:02";
```

Comment:
Can we define this 2 predefined switchId as SwitchId objects? It will eliminate requirements in most conversation inside test methods.

Reply:
Yes, we can. I will fix it.